phoenix-core/src/main/java/org/apache/calcite/sql/SqlOptionNode.java

```diff
@@ -38,12 +41,10 @@ public SqlOptionNode(SqlParserPos pos, SqlIdentifier key, SqlLiteral literal) {
             familyName = key.names.get(0);
             propertyName = key.names.get(1);
         }
-        final Object v = SqlLiteral.value(literal);
-        if (v instanceof NlsString) {
-            value = ((NlsString) v).toString();
-        } else {
-            value = v;
-        }
+
+        PhoenixRelImplementor
+                implementor = new PhoenixRelImplementorImpl(new RuntimeContextImpl());
```

Comment:
Looks like we need a "dummy" RuntimeContext for this kind of "dry run" implementor. I'll do this part when I commit it.